karma.config.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function (config) {
+
+  return config.set({
+    basePath: '.',
+
+    browsers: [ 'Chrome' ],
+
+    frameworks: [ 'mocha' ],
+
+    files: [
+      './test/**/*.test.js*',
+      './test/index.html'
+    ],
+
+    preprocessors: {
+      './test/**/*.test.js*': ['webpack'],
+      '**/*.html': ['html2js']
```

Comment:
Let's me bring in an html file as a test fixture:
https://github.com/vigetlabs/ca11y/pull/23/files#diff-2bd6d9916278bc10d8449bf85ca11c05R8

Basically just turns the html file into a string and sticks it on `window.__html__`.